apparmor.d/profiles-m-r/rustdesk

```diff
@@ -0,0 +1,143 @@
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_path} = /{,usr/}{,local/}bin/rustdesk
+profile rustdesk @{exec_path} {
+  include <abstractions/base>
+  include <abstractions/nameservice-strict>
+  include <abstractions/openssl>
+  include <abstractions/X-strict>
+  include <abstractions/fonts>
+  include <abstractions/fontconfig-cache-read>
+  include <abstractions/freedesktop.org>
+  include <abstractions/dbus-session-strict>
+  include <abstractions/dbus-accessibility-strict>
+  include <abstractions/dbus-gtk>
+
+  network inet dgram,
+  network inet6 dgram,
+  network inet stream,
+  network inet6 stream,
+  network netlink raw,  # discovery
+
+  @{exec_path} mrix,
+
+  /{,usr/}bin/ps       rPx,
+  /{,usr/}bin/whoami   rPx,
+  /{,usr/}bin/loginctl rPx,
+  /{,usr/}bin/curl     rix,
+
+  /{,usr/}bin/python3.[0-9]* rCx -> python,
+
+  dbus (send) bus=accessibility path=/org/a11y/atspi/accessible/root
+       interface=org.a11y.atspi.Socket
+       member=Embed
+       peer=(name=org.a11y.atspi.Registry),
+
+  dbus (send) bus=accessibility path=/org/a11y/atspi/registry
+       interface=org.a11y.atspi.Registry
+       member=GetRegisteredEvents
+       peer=(name=org.a11y.atspi.Registry),
+
+  dbus (send) bus=accessibility path=/org/a11y/atspi/registry/deviceeventcontroller
+       interface=org.a11y.atspi.DeviceEventController
+       member={GetKeystrokeListeners,GetDeviceEventListeners}
+       peer=(name=org.a11y.atspi.Registry),
+
+  dbus (receive) bus=accessibility path=/org/a11y/atspi/accessible/root
+       interface=org.freedesktop.DBus.Properties
+       member=Set
+       peer=(name=:*),
+
+  owner /tmp/[rR]ust[dD]esk/{,**} rw,
+
+  owner @{user_share_dirs}/logs/[rR]ust[dD]esk/{,**} rw,
+  owner @{user_config_dirs}/[rR]ust[dD]esk/{,**} rw,
+
+  owner @{run}/user/@{uid}/at-spi/bus{,_[0-9]*} rw,
+
+  @{sys}/devices/system/cpu/cpufreq/policy[0-9]*/scaling_{cur,min,max}_freq r,
+
+        @{PROC}/uptime r,
+  owner @{PROC}/@{pid}/cgroup r,
+  owner @{PROC}/@{pid}/cmdline r,
+
+  # pulse
+  /dev/shm/ r,
+  /etc/pulse/client.conf r,
+  /etc/pulse/client.conf.d/{,*} r,
+  owner @{run}/user/@{uid}/pulse/ r,
+  owner @{run}/user/@{uid}/pulse/native rw,
+  owner @{user_config_dirs}/pulse/ rw,
+  owner @{user_config_dirs}/pulse/cookie rwk,
+
+  # gtk-tiny
+  /etc/gtk-3.0/settings.ini r,
+  /usr/share/themes/*/gtk-3.0/gtk.css r,
+
+  # file transfer
+  owner @{HOME}/ r,  # fails otherwise
+  owner @{HOME}/[rR]ust[dD]esk/{,**} rw,
+
+  # file_inherit, X-strict
+  owner @{HOME}/.xsession-errors w,
+
+  # excessive?
+  deny @{PROC} r,
+#  @{PROC} r,
+#  capability sys_ptrace,
+#  ptrace (read),
+#  owner @{PROC}/@{pid}/stat r,
+#  owner @{PROC}/@{pid}/environ r,
+#  owner @{PROC}/@{pid}/io r,
+#  owner @{PROC}/@{pid}/task/ r,
+#  owner @{PROC}/@{pid}/task/@{tid}/stat r,
+#  owner @{PROC}/@{pid}/task/@{tid}/io r,
+#  owner @{PROC}/@{pid}/task/@{tid}/status r,
+
+  profile python {
```

Comment:
Nice idea to put the python part of a program done in rust in its own subprofile.
Do you know the purpose of this python segment? 

These lines make me think that could be they update system:
```
   /usr/share/dpkg/cputable r,
    deny /etc/apt/{,**} r,
```

In this case `/etc/apt/` might be needed

Reply:
>Do you know the purpose of this python segment?

This is somehow related to file transfer, and not essential.

>These lines make me think that could be they update system:

Then we shouldn't allow this access to a remote control program. I'm thinking it's some python artifact, whether related to Xubuntu or my specific installation, I'm not sure yet.

Reply:
That is not python related, this is dpkg resources (See the apt-common abstraction).

> This is somehow related to file transfer, and not essential.

Ok, let's keep it like this for now, might need more resources later (like network access)